python/in_toto_attestation/v1/statement.py

```diff
@@ -0,0 +1,36 @@
+#  Wrapper class for in-toto attestation Statement protos.
+
+import in_toto_attestation.v1.statement_pb2 as spb
+from in_toto_attestation.v1.resource_descriptor import ResourceDescriptor
+
+STATEMENT_TYPE_URI = 'https://in-toto.io/Statement/v1'
+
+class Statement:
+    def __init__(self, s=None):
+        self.pb = spb.Statement()
```

Comment:
Should this set self.pb.type to STATEMENT_TYPE_URI so that users don't have to?

Reply:
done.